utils/update_sc_scheduled_jobs.py

```diff
@@ -0,0 +1,35 @@
+# Description: Update the status of a scheduled job in the Service Catalogue
+import os
+from time import sleep
+from datetime import datetime
+from classes.service_catalogue import ServiceCatalogue
+import globals
+
+log_level = os.environ.get('LOG_LEVEL', 'INFO').upper()
+
+class Services:
```

Comment:
does this module reflect one or more jobs? if so would a better name for this be `Jobs`?

Reply:
or if it really represents a single job (you call it `update_sc_scheduled_job` above), would a better name be: 
`Update_Sc_Scheduled_job`?

is maybe the fact that it's scheduled is an implementation detail it knows nothing about? 
`UpdateServiceCatalogueJob` ??